pkg/cmd/pr/list/list.go

```diff
@@ -116,6 +118,19 @@ func listRun(opts *ListOptions) error {
 		return err
 	}
 
+	argv, err := shlex.Split(opts.Search)
+	if err != nil {
+		return err
```

Comment:
Let's not abort the whole command if `shlex.Split` failed. If splitting the query string into arguments isn't possible, that's still not reason enough to abort the whole command. The query might still be valid to ElasticSearch.